README.md

```diff
@@ -1,4 +1,185 @@
+  </ul>
+</details>
+
+<details>
+  <summary>Server</summary>
+  <ul>
+  </ul>
+</details>
+
+<details>
+<summary>Database</summary>
+  <ul>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+> The application delivers information about the green energy, and it has the following features.
+
+- Mobile first approach and responsive
+- Section management
+- Dynamic javascript data binding
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+> Links related to the project.
+
+- [Live Demo Link](https://habasm.github.io/habasm-capstone1/)
+- [Watch Video demo](https://www.loom.com/share/eb2708584b6347abafcb6e0a04158bd0/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running follow these simple steps:
+
+1. Go to the [repository page](https://github.com/habasm/habasm-capstone1/tree/fr-branch).
+2. Press the "Code" button and copy the link.
+3. Clone it using git command `git clone https://github.com/habasm/habasm-capstone1.git`.
+
+### Prerequisites
+
+In order to run this project you need:
+- Just copy the contents and utilize the codes.
+
+### Setup
+
+Clone this repository to your desired folder:
+
+
+Example commands:
+
+```sh
+  cd habasm-capstone1
+  git clone https://github.com/habasm/habasm-capstone1.git
+```
+
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+> Mention all of the collaborators of this project.
+
+👤 **Author1**
+
+- GitHub: [@habasm](https://github.com/habasm)
+- Twitter: [@twitterhandle](https://twitter.com/AsimareHabitamu)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/AsimareHabitamu)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+> IN the next, the following features will be added.
+
+- Guest comments section
+- Database integration
+- Layout validations
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/habasm/habasm-capstone1/issues).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+- This project to be successful #Microverse plays great roles. And my family and friends allows me to express my feelings while doing this.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to thank Microverse for this project to be real here.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
```

Comment:
- [ ] **[OPTIONAL]:** Since this design is under a Creative Commons license please make sure you give the respective credit to the author 🎨.

![image](https://user-images.githubusercontent.com/77069011/203575435-b2671719-d73c-43c4-8462-8eb32dda3ec3.png)
  
⚠️ **I'm sorry the previous reviewer didn't point out this, but it is VERY IMPORTANT** ⚠️ 

Please consider doing this! 🤓 🛠️